### ISSUE TITLE
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]

### ISSUE THREAD

**Issue by cereschen** [BuilderIO/jsx-lite#77]
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main resolved in C:\Users\GT-N02\AppData\Roaming\npm\node_modules\@jsx-lite\cli\node_modules\@jsx-lite\core\package.json 

os: win10
node: 12.18.2 15.11.0

**Comment by steve8708** [comment]
Hi @cereschen,

We need more information to look into your error. In particular, additional context such as what were the steps to reproduce?

**Comment by cereschen** [comment]
npm install -g @jsx-lite/cli
cd D:\test
create file: D:\test\test.tsx
paste:
```ts
import { useState, Show, For } from '@jsx-lite/core';

export default function MyComponent(props) {
  const state = useState({
    newItemName: 'New item',
    list: ['hello', 'world'],
    addItem() {
      state.list = [...state.list, state.newItemName];
    },
  });

  return (
    <div>
      <Show when={props.showInput}>
        <input
          value={state.newItemName}
          onChange={(event) => (state.newItemName = event.target.value)}
        />
      </Show>
      <div css={{ padding: '10px' }}>
        <button onClick={() => state.addItem()}>Add list item</button>
        <div>
          <For each={state.list}>{(item) => <div>{item}</div>}</For>
        </div>
      </div>
    </div>
  );
}
```

run  jsx-lite compile --to=vue test.tsx


I didn't have this error in Node 12.16.1 today but later versions will still report the error

**Comment by cereschen** [comment]
$ jsx-lite compile --to=vue my-file.lite.jsx
C:\Users\GT-N02\AppData\Roaming\nvm\v15.11.0\node_modules\@jsx-lite\cli\node_modules\gluegun\build\index.js:13
    throw up;
    ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in C:\Users\GT-N02\AppData\Roaming\nvm\v15.11.0\node_modules\@jsx-lite\cli\node_modules\@jsx-lite\core\package.json
    at new NodeError (node:internal/errors:329:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:337:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:562:3)
    at resolveExports (node:internal/modules/cjs/loader:473:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:513:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:910:27)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (C:\Users\GT-N02\AppData\Roaming\nvm\v15.11.0\node_modules\@jsx-lite\cli\build\commands\build.js:57:14) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

**Comment by Dkendal** [comment]
Hey @cereschen this should be fixed in @jsx-lite/cli version 0.0.3.
